src/WebJobs.Extensions.DocumentDB/Bindings/DocumentDBItemValueBinder.cs

```diff
@@ -17,11 +18,11 @@ internal class DocumentDBItemValueBinder<T> : IValueBinder where T : class
         private ParameterInfo _parameter;
         private DocumentDBContext _context;
         private JObject _originalItem;
-        private string _id;
+        private DocumentDBItemBindingContext _bindingContext;
 
-        public DocumentDBItemValueBinder(ParameterInfo parameter, DocumentDBContext context, string id)
+        public DocumentDBItemValueBinder(ParameterInfo parameter, DocumentDBContext context, DocumentDBItemBindingContext bindingContext)
         {
-            _id = id;
+            _bindingContext = bindingContext;
```

Comment:
Fixed.

---
In reply to: [59795439](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/63#discussion_r59795439) [](ancestors = 59795439)